doc/sphinx/language/core/definitions.rst

```diff
@@ -31,6 +31,8 @@ for :n:`let @ident := fun {+ @binder} => @term__1 in @term__2`.
    single: ... <: ...
    single: ... <<: ...
 
+.. _type-cast:
+
 Type cast
 ---------
```

Comment:
This section could also benefit from a reference to the section on `vm_compute` and `native_compute`. Currently, the explanations on `<:` and `<<:` are a bit dry.

Reply:
I'll improve it.

FWIW, "dry" is not the word I'd use here.  Perhaps "incomplete".  Here are the relevant definitions; IMHO the first mainly means "boring" (e.g. law books, many textbooks):

2 bare or lacking adornment: 

* "the dry facts" unexciting; dull.
"by current tastes the text is dry"
**Similar**: bare simple basic fundamental stark naked bald cold hard straightforward unadorned unembellished dull uninteresting boring unexciting tedious tiresome wearisome dreary monotonous dry as dust arid unimaginative sterile flat bland insipid lackluster stodgy colorless lifeless prosaic run-of-the-mill humdrum mundane commonplace workaday quotidian routine vapid stiff leaden wooden deadly samey dreich
**Opposite**: embellished interesting lively

* unemotional, undemonstrative, or impassive.
"Ralph gave me a dry, silent wave"

Reply:
The <: and <:: forms both apply a cast AND indicate whether to check with vm_compute or native_compute?  If you don't actually need to cast the term, it seems unfriendly that you have to give the `@type` just to mark it for vm_compute or native_compute, though I suppose in practice you would always rely on those tactics to add the <: and <::.

Reply:
```
:n:`@term10 <: @type` specifies that the virtual machine will be used
to type check that :n:`@term10` has type :n:`@type` (see :tacn:`vm_compute`).

:n:`@term10 <<: @type` specifies that compilation to OCaml will be used
to type check that :n:`@term10` has type :n:`@type` (see :tacn:`native_compute`).
```

Reply:
Indeed. I think it is almost always added by one of these tactics as opposed to manually added.